### DISCUSSION TITLE
Bticino K4027C

### DISCUSSION THREAD

**Discussion by Empathy3286** [Koenkk/zigbee2mqtt#16923] [Devices]
Hi,

I've got a bunch of "Bticino K4027 Shutter SW with level control" in the house.
I can successfully pair them with home assistant via zigbee2mqtt.
Unfortunately i can't control the exact level of the shutter. it's either open or closed (i can also stop it midway)

Anyway, that's not the main problem.
The shutter control has a little led light, and it keeps blinking green. It's a bit annoying.
Can i turn off the blinking?

Thanks

**Comment by altrdev** [comment]
Yeah I have the same problem. Bticino App offers a calibration procedure for percentage control.
I don't know if is possible introduce calibration in zigbee2mqtt

Same problem with blinking led after pairing

**Reply by developerGM** [reply]
Did you solve it?
I can't set the percentage too

**Reply by jafar-atili** [reply]
You should hold the up and down for a few seconds and it will start the calibration process, this can take few minutes to complete

**Comment by MP14588** [comment]
Hi,

same problem here. I have four Bticino K4027. Only one shows the percentage after pairing with zigbee2mqtt, the other three have the same problems as yours: flashing green LED after pairing and no position control. It is written in the z2mqtt documentation that some older firmware devices cannot support position control (0, 50 and 100% only). Has anyone found a solution to this? Even updating the firmware doesn't work for me.
P.s. After pairing, no calibration is activated by pressing the up and down buttons simultaneously.

Thank you!

**Comment by xraver** [comment]
I've tested today with NT4027C. no issue found.
if you press both buttons (up and down) for 5 seconds, orange leds starts to toggling and the procedure starts.

**Comment by Crystian182** [comment]
what if the calibration does not match the reality? i mean, setting the percentage at 2% the shutter is completely closed

**Reply by xraver** [reply]
with 2% what happens if you press the close button?

**Reply by Crystian182** [reply]
well, I can clearly hear the motor working to close the remaining part, but visually, the shutter is already closed at 2%

i would like to manually calibrate the shutter depending on how i perceive it

**Reply by Crystian182** [reply]
i've just found out someone describing how to manually calibrate the shutter [here](https://github.com/Koenkk/zigbee-herdsman-converters/pull/6333#issuecomment-1790548665)

i don't know if it works, i can give it a try

**Reply by xraver** [reply]
i have the same behaviour with shelly.
in general the motors stop working based on calibration. if the calibration of the motor is not precise this lead to issues (fine-corsa).
in additional it's necessary to take into account the calibration of the smart device: this is another story. based on the consumption the devices understand when the motor stop working. this happens when is complitely open or closed.
